### PR TITLE
Media is now served over HTTPS by default 

### PR DESCRIPTION
**Pull request by logan-anderson**

### CODE REVIEW

**Review by Enigmatical — APPROVED** [review]

File: packages/next-tinacms-cloudinary/src/handlers.ts [hunk 1]
```diff
@@ -164,18 +175,34 @@ async function deleteAsset(req: NextApiRequest, res: NextApiResponse) {
...
-
-  return {
-    id: file.public_id,
-    filename,
-    directory,
-    src: file.url,
-    previewSrc: transformCloudinaryImage(file.url, 'w_75,h_75,c_fill,q_auto'),
-    type: 'file',
+    const sel = useHttps ? ('secure_url' as const) : ('url' as const)
+    console.log({ sel })
```
Comment (Enigmatical) [inline packages/next-tinacms-cloudinary/src/handlers.ts:190]: Look goods 👍 Let's remove this `console.log()` here, though.

Comment (logan-anderson) [reply packages/next-tinacms-cloudinary/src/handlers.ts]: Good catch!

File: packages/tinacms/src/tina-cms.tsx [hunk 2]
```diff
@@ -268,8 +268,12 @@ export const TinaCMSProvider2 = ({
     props.query
   }
 
+  const validOldSetup =
+    new Boolean(props?.isLocalClient) ||
+    (new Boolean(props?.clientId) && new Boolean(props?.branch))
+
   // branch & clientId are still supported, so don't throw if they're provided
-  if (!props.apiURL && !(props?.clientId || props?.isLocalClient)) {
+  if (!props.apiURL && !validOldSetup) {
```
Comment (jeffsee55) [inline packages/tinacms/src/tina-cms.tsx:276]: What was the combination of props that this failed for? Was it that you could forget to include `branch`?

**Review by jeffsee55 — APPROVED** [review]
Just a quick question

**Pull request merged**